src/Network/Network/help/Get-AzVirtualNetworkGatewayVpnclientConnectionHealth.md

```diff
@@ -0,0 +1,146 @@
+﻿---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/az.network/get-azvirtualnetworkgatewayvpnclientconnectionhealth
+schema: 2.0.0
+---
+
+# Get-AzVirtualNetworkGatewayVpnclientConnectionHealth
+
+## SYNOPSIS
+Get the list of vpn client connection health of an Azure virtual network gateway for per vpn client connection
+
+## SYNTAX
+
+```
+Get-AzVirtualNetworkGatewayVpnclientConnectionHealth -VirtualNetworkGatewayName <String> -ResourceGroupName <String> -InputObject <PSVirtualNetworkGateway> -ResourceId <ResourceId>
+ [-AsJob] [<CommonParameters>]
+```
+
+## DESCRIPTION
+List  all connected vpn client connection health. This includes:
+vpnConnectionId
+vpnConnectionDuration
+vpnConnectionTime
+publicIpAddress
+privateIpAddress
+vpnUserName
+maxBandwidth
+egressPacketsTransferred
+egressBytesTransferred
+ingressPacketsTransferred
+ingressBytesTransferred
+maxPacketsPerSecond
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> Get-AzVirtualNetworkGatewayVpnclientConnectionHealth -ResourceGroupName resourceGroup -VirtualNetworkGatewayName gatewayName
```

Comment:
Example isn't using valid PowerShell. If `resourceGroup` and `gatewayName` are variables, they would be accessed via `$resourceGroup` and `$gatewayName`. Also, cmdlet name should be Get-AzVirtualNetworkGatewayVpn**C**lientConnectionHealth. And, the example output is not from PowerShell (it wouldn't display anything like that).